libs/build_esmf.sh

```diff
@@ -113,7 +113,8 @@ case $MPI in
     export ESMF_COMM=${STACK_esmf_comm:-"mpich3"}
     ;;
   cray-mpich )
-    export ESMF_COMM=${STACK_esmf_comm:-"mpi"}
+    export ESMF_OS="Linux"
```

Comment:
This one is tricky and should not be necessary.
If it is indeed necessary, please replace with:
```
export ESMF_OS=${STACK_esmf_os:-$host}
```

Reply:
Good suggestion, the export ESMF_OS=${STACK_esmf_os:-$host} works